### DISCUSSION TITLE
Settings for Clear Background – No Blur Effect. Is it possible to maintain consistent clothing across different poses?

### DISCUSSION THREAD

**Discussion by Barca98** [lllyasviel/Fooocus#3714] [Q&A]
Hi, I would like to create images where the background remains sharp instead of blurred. I have tried a few settings, but I still can't achieve the desired effect. Could someone help me with the correct settings to ensure the background stays clear? Any suggestions would be greatly appreciated! Is it also possible to create a series of images where the character has the same clothing but adopts different poses? I’ve tried adjusting settings like models, styles, Guidance Scale, and Image Sharpness to different levels.
I expect the background to be clear, and the character will be well integrated with the environment

I'm attaching an image that shows the result I got with my current settings:

1. What I am creating: full body photo of a girl with medium brown hair and brown eyes
she is leaning against a building, wearing high heels, a black skirt, a white top and a black leather jacket. background walkway to the park which has many details that are not blurry 
2. Negative Prompt: unrealistic, saturated, high contrast, big nose, painting, drawing, sketch, cartoon, anime, manga, render, CG, 3d, watermark, signature, label, nude, nsfw, text, letters, too many feet, too many fingers, long neck, 2 heads, duplicate, abstract, disfigured, deformed, toy, figure, framed, disfigured, bad art, deformed, poorly drawn, extra limbs, weird colors, 2 heads, elongated body, cropped image, out of frame, draft, deformed hands, twisted fingers, double image, malformed hands, multiple heads, extra limb, ugly, poorly drawn hands, missing limb, cut-off, over satured, grain, lowères, bad anatomy, poorly drawn face, mutation, mutated, floating limbs, disconnected limbs, out of focus, long body, disgusting, extra fingers, groos proportions, missing arms, mutated hands, cloned face, missing legs,
3. Styles: Fooocus V2, Fooocus Sharp, Photo Iphone Photographic
4. Models: juggernautXL_v8Rundiffusion, Lora: 1sd_xl_offset_example - Weight 0,25
5. Guidance Scale: 2 
6. Image Sharpness: 12

![2024-10-27_17-39-00_7944](https://github.com/user-attachments/assets/de1dca23-1995-4f4d-a1d5-c90f18cc108e)

**Comment by danort92** [comment]
Up
